src/main/java/com/uid2/shared/secure/gcpoidc/PolicyValidator.java

```diff
@@ -30,14 +32,24 @@ public class PolicyValidator implements IPolicyValidator {
 
     private static final List<String> REQUIRED_ENV_OVERRIDES = ImmutableList.of(
             ENV_ENVIRONMENT,
-            ENV_OPERATOR_API_KEY_SECRET_NAME,
-            ENV_CORE_ENDPOINT,
-            ENV_OPT_OUT_ENDPOINT
+            ENV_OPERATOR_API_KEY_SECRET_NAME
     );
 
     private static final Map<Environment, List<String>> OPTIONAL_ENV_OVERRIDES_MAP = ImmutableMap.of(
-            Environment.Integration, ImmutableList.of()
+            Environment.Production, ImmutableList.of(
```

Comment:
Do we want to override for production?

Reply:
Yes, we do, so that the url is put into the manifest that is signed and passed for attestation. We can then in Core validate that the Core URL is the one we expect.